### PR TITLE
3.4.0 Release

### PR DESCRIPTION
**Pull request by nickva**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -544,36 +544,36 @@ <h2 class="icon icon-download">Download CouchDB 3.3.3</h2>
...
             </span>
           </li>
           <li>
             <a href="https://neighbourhood.ie/download-apache-couchdb-mac/" class="type">
               macOS
             </a>
             <span class="info">
-              Erlang/OTP 26.0.2 | Version 3.3.3 |
-              <a href="https://docs.couchdb.org/en/latest/whatsnew/3.3.html#version-3-3-3" class="release">Release Notes</a>
+              Erlang/OTP 26.0.2 | Version 3.4.0 |
```
Comment (nickva) [inline index.html:569]: Will need to update around the time of the announcement (cc: @janl)

File: index.html [hunk 2]
```diff
@@ -544,36 +544,36 @@ <h2 class="icon icon-download">Download CouchDB 3.3.3</h2>
...
             </span>
           </li>
           <li>
             <a href="https://neighbourhood.ie/download-apache-couchdb-win/" class="type">
               Windows (x64)
             </a>
             <span class="info">
-              Erlang/OTP 24.3.4.17 | Version 3.3.3 |
-              <a href="https://docs.couchdb.org/en/latest/whatsnew/3.3.html#version-3-3-3" class="release">Release Notes</a>
+              Erlang/OTP 25.3.2.13 | Version 3.4.0 |
```
Comment (nickva) [inline index.html:560]: Will need to update around the time of the announcement (cc: @big-r81)

Comment (big-r81) [reply index.html:560]: Yup, that is the included Erlang/OTP version.

**Comment by nickva** [comment]
Closing in favor of the 3.4.1 release

**Pull request closed without merging**